### PR TITLE
nits

### PR DESCRIPTION
**Pull request by okjodom**
a bunch of unrelated nitpicks that have me renaming things :)

### CODE REVIEW

File: gateway/ln-gateway/tests/integration_tests.rs [hunk 1]
```diff
@@ -602,7 +602,7 @@ async fn test_gateway_register_with_federation() -> anyhow::Result<()> {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn test_gateway_cannot_pay_expired_invoice() -> anyhow::Result<()> {
-    single_gateway_test(
+    single_federation_test(
```
Comment (dpc) [inline gateway/ln-gateway/tests/integration_tests.rs:605]: **thought:** "single" here is confusing. Is it single test, single federation... or single peer in a federation? :D

**Review by dpc — APPROVED** [review]
Please have someone else voice an opinion, but I guess LGTM.

File: gateway/ln-gateway/tests/integration_tests.rs (hunk 1 above)
Comment (okjodom) [reply gateway/ln-gateway/tests/integration_tests.rs:605]: For context, this test scenario involves the gateway and just one federation. The alternative test scenario where a gateway interacts with many federations is named `multi_federation_test`

**Review by m1sterc001guy — APPROVED** [review]

**Comment by m1sterc001guy** [comment]
I like the renames

**Pull request merged**